### ISSUE TITLE
NullPointerException: 'void androidx.recyclerview.widget.RecyclerView.o0(int)'

### ISSUE THREAD

**Issue by nostra13** [helpcrunch/android-sdk-demo#22]
Crashed on production:

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.recyclerview.widget.RecyclerView.o0(int)' on a null object reference
       at com.helpcrunch.library.e.b.c.c$q.run(c.java:349)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6762)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

`implementation "com.helpcrunch:chat-sdk:3.0.7"`

**Android:** 5.0.1, 5.0.2, 9, 10
**Devices:** Mi A1,  Redmi 7A, Redmi Note 7, Galaxy A3, V831W, Galaxy A51, Galaxy S10, HONOR View20

**Comment by TalbotGooday** [comment]
Thanks for the report!

**Comment by TalbotGooday** [comment]
Fixed in `3.0.8-alpha.01`
